src/ExecuteTimeWidget.ts

```diff
@@ -3,10 +3,9 @@ import { JSONExt, JSONObject } from '@lumino/coreutils';
 import { NotebookPanel, INotebookTracker } from '@jupyterlab/notebook';
 import { ISettingRegistry } from '@jupyterlab/settingregistry';
 import {
-  IObservableJSON,
   IObservableList,
-  IObservableUndoableList,
 } from '@jupyterlab/observables';
+import { CellList } from '@jupyterlab/notebook/lib/celllist';
```

Comment:
You could use `import type` here, and `/lib` would ideally be dropped (if it is not exported, it should be fixed upstream).

Reply:
Yep, it's not exported. I can make a PR upstream.

Reply:
Upstream PR: https://github.com/jupyterlab/jupyterlab/pull/14185

Reply:
Upstream PR is merged, so I'll test this locally now.